docs/source-app/api_reference/api/lightning_app.core.LightningFlow.rst

```diff
@@ -1,6 +1,4 @@
-﻿:orphan:
-
-.. role:: hidden
```

Comment:
This file shouldn't be commited.

Reply:
ok, why? they seem to be auto-generated and basically stubs?